cmd/gcp-controller-manager/node_annotator.go

```diff
@@ -42,6 +43,8 @@ import (
 // InstanceIDAnnotationKey is the node annotation key where the external ID is written.
 const InstanceIDAnnotationKey = "container.googleapis.com/instance_id"
 
+const lastAppliedLabelsKey = "gcp-controller-manager.kubernetes.io/last-applied-node-labels"
```

Comment:
let's sync on the annotation key.

Reply:
security.gke.io/last-applied-node-labels according to naming convention